docs/content/about/intro.md

```diff
@@ -23,7 +23,7 @@ Open Infrastructure Labs also has a mailing list: Openinfralabs http://lists.ope
 You can view a dashboard in the OI Labs github to see an overview of what’s happening https://github.com/orgs/open-infrastructure-labs/projects/2
 
 
-### Bring your project
+## Bring your project
 
 To get connected and investigate bringing your project to the MOC, you can join the general mailing list at https://listman.redhat.com/mailman/listinfo/operate-first and introduce it.
```

Comment:
List should be https://lists.operate-first.cloud/archives/list/community@lists.operate-first.cloud/

Reply:
updated